### ISSUE TITLE
Status of the project

### ISSUE THREAD

**Issue by bergerx** [redhat-cop/namespace-configuration-operator#48]
Hi, this morning i've played a bit with the operator and so far this one is the closest tool that we currently need, we'd like to use and contribute to the project rather than creating an internal tool from scratch, but I have some open questions in mind:
* We are not using openshift, would you be willing to accept PRs for adding features as if this is an independent open source project, or, is this tool being explicitly designed for openshift and will stay that way? Also seeing User and Group object added in the 0.2.0 makes me think more like "this seems to be openshift specific".
* Are there any existing users of this project or is this just some sandbox/PoC project to be deprecated after proving the approach/point?
* Are you willing to care about external contributors (like us) e.g. include/notify them for changes, have a changelog. I know its still pre-v1 and also the crd version is still v1alpha1 and we should expect breaking changes at any point at least before CRDs are beta or the operator is v1. But I'm more curious about where are you with the implementation and how much more breaking changes we should expect before operator/CRDs reach a relatively stable state. A recent example i see:
  * breaking change inside the v1alpha1 version of the CRD. In v0.1.0 had a `NamespaceConfig.spec.resources` and v0.2.0 now has that field removed and replaced with `NamespaceConfig.spec.templates.objectTemplate`, quite a big structural change in the crd.

Also it would be helpful if you include such information into the readme.

**Comment by raffaelespazzoli** [comment]
@bergerx 
this project is open for contribution and should be able to run on any kubernetes cluster. I recently made a change to make sure that happened. That said kubernetes does not seem to have the concept of user and groups, so I didn't know how to map those functionalities on  kube. For now they are disabled. Any design/implementation suggestions here would be welcome.
To answer you other questions: yes we have several users mostly in the openshift space (including users that use this in production), and you are right we have reached a level of adoption for this project where we should start publishing release notes.
As far as the CRD maturity, I wanted to see the reactions after this breaking change. maybe we are getting to the point where we can promote it to v1beta1, but I am not sure yet.
